cs/ccxt/ws/Exchange.WsBridge.cs

```diff
@@ -60,7 +60,25 @@ public async virtual Task loadOrderBook(WebSocketClient client, object messageHa
         object tries = 0;
         try
         {
-            var stored = getValue(this.orderbooks, symbol) as ccxt.pro.OrderBook;
+            var storedRaw = getValue(this.orderbooks, symbol);
+            ccxt.pro.IOrderBook stored = null;
+            if (storedRaw is ccxt.pro.OrderBook)
+            {
+                stored = storedRaw as ccxt.pro.OrderBook;
+            }
+            else if (storedRaw is ccxt.pro.IndexedOrderBook)
+            {
+                stored = storedRaw as ccxt.pro.IndexedOrderBook;
+            }
+            else if (storedRaw is ccxt.pro.CountedOrderBook)
+            {
+                stored = storedRaw as ccxt.pro.CountedOrderBook;
+            }
```

Comment:
I~m not sure this cast will have any impact in practice

Reply:
it does, as mentioned in previous days, casting to .OrderBook ends up with null. so, it should be casted to correct type

Reply:
this update fixed the exceptions being thrown before this.